src/components/position.js

```diff
@@ -1,8 +1,8 @@
-var coordinatesMixin = require('../utils/coordinates').componentMixin;
 var registerComponent = require('../core/component').registerComponent;
-var utils = require('../utils/');
 
-module.exports.Component = registerComponent('position', utils.extend({
+module.exports.Component = registerComponent('position', {
+  type: 'coordinate',
```

Comment:
I would say vec3 so it matches the name in shaders glsl world. We will use the same type for custom shaders.

> On Jan 8, 2016, at 4:33 PM, Kevin Ngo <notifications@github.com> wrote:
> 
> In src/components/position.js:
> 
> >  
> > -module.exports.Component = registerComponent('position', utils.extend({
> > +module.exports.Component = registerComponent('position', {
> > +  type: 'coordinate',
> Okay, I had a long train of thought, and I think that is fine.
> 
> What do you think about vector3 or vec3 instead of v3? three.js is not using v3 anymore, it sounds like "version 3", and Googling v3 or 3d v3 turns up nothing. Googling vector3 is pretty useful though.
> 
> —
> Reply to this email directly or view it on GitHub.
>